staging/src/k8s.io/apiserver/pkg/authentication/request/bearertoken/bearertoken_test.go

```diff
@@ -184,6 +184,13 @@ func TestBearerToken(t *testing.T) {
 			ExpectedErr:                  true,
 			ExpectedAuthorizationHeaders: []string{"Bearer 123"},
 		},
+		"valid bearer token with a space": {
```

Comment:
you can add a dummy warning recorder to the context in the dummy request passed into the unit test (see dummyRecorder in pkg/registry/batch/job/storage/storage_test.go for an example) and verify the warning was added here

Reply:
Done!